CONTRIBUTING.md

```diff
@@ -21,6 +21,78 @@ To speed up the review process, please use the provided pull request template an
 
 The pull request template includes areas to explain the changes, and a checklist with boxes for code style, testing, and documenttation.
 
+## Commit Message Format
+
+Each commit message consists of a **header**, a **body**, and a **footer**. The header has a special format that includes a **type**, a **scope**, and a **subject**:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+The **header** is mandatory, and the **scope** of the header is optional. The commit message lines should not exceed 100 characters in length.
+
+### Type
+
+The type must be one of the following:
+
+- **feat**: A new feature
+  - Increments the minor version number (e.g., 1.0.0 -> 1.1.0)
+
+- **fix**: A bug fix
+  - Increments the patch version number (e.g., 1.0.0 -> 1.0.1)
+
+- **docs**: Documentation-only changes
+  - Does not affect the version number
+
+- **style**: Changes that do not affect the code's meaning (e.g., white-space, formatting)
+  - Does not affect the version number
+
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+  - Does not affect the version number
+
+- **perf**: A code change that improves performance
+  - Does not affect the version number
+
+- **test**: Adding missing or correcting existing tests
+  - Does not affect the version number
+
+- **chore**: Changes to the build process, auxiliary tools, or libraries
+  - Does not affect the version number
+
+### Scope
+
+The scope specifies the location of the commit change. For example: `$location`, `$browser`, `$compile`, `$rootScope`, `ngHref`, `ngClick`, `ngView`, etc. Use `*` when the change affects more than a single scope.
```

Comment:
@Mustaballer these scopes apply to angular, but not so much to openadapt. Can you please provide some examples?